semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/rpc/RpcSemanticConventionUtils.java

```diff
@@ -102,12 +102,29 @@ public static boolean isRpcTypeGrpcForOTelFormat(Event event) {
             event, getAttributeKeysForRpcService()));
   }
 
+  public static Optional<String> getRpcMethod(Event event) {
```

Comment:
I think, this method is for this - https://github.com/hypertrace/data-model/blob/main/data-model/src/main/avro/eventfields/rpc/Rpc.avdl, right?

Reply:
@kotharironak  Mentioned over here. 
https://github.com/Traceableai/trace-enricher/pull/275/files#r662025316